resources/runner

```diff
@@ -237,7 +237,7 @@ set -euo pipefail
 
 snakemake --latency-wait 120 -s "$3/workflow/Snakefile" -d "$3" \\
   --use-singularity --singularity-args "'-B $4'" --configfile="$3/config.json" \\
-  --printshellcmds --cluster-config "$3/config/cluster.json" \\
+  --printshellcmds --cluster-config "$3/cluster.json" \\
   --cluster "${CLUSTER_OPTS}" --keep-going --restart-times 3 -j 500 \\
```

Comment:
Can you change CLUSTER_OPTS like [this](https://github.com/CCBR/RENEE/blob/474884f5755316bc5da5f8ce83fce85f472bdf2f/resources/runner#L163) and later add `--gres` like [this](https://github.com/CCBR/RENEE/blob/474884f5755316bc5da5f8ce83fce85f472bdf2f/resources/runner#L180) .. this is required as gres is not available on FRCE. Also compare other things with the v2.2 RENEE runner

Reply:
both `runner` and `config.cluster.json` has been updated.